pkg/apis/trust/v1alpha1/types_bundle.go

```diff
@@ -95,13 +95,21 @@ type BundleTarget struct {
 	// ConfigMap is the target ConfigMap in Namespaces that all Bundle source
 	// data will be synced to.
 	ConfigMap *KeySelector `json:"configMap,omitempty"`
+	// AdditionalFormats specifies any additional formats to write to the configmap
+	// +optional
+	AdditionalFormats *AdditionalFormats `json:"additionalFormats,omitempty"`
 
 	// NamespaceSelector will, if set, only sync the target resource in
 	// Namespaces which match the selector.
 	// +optional
 	NamespaceSelector *NamespaceSelector `json:"namespaceSelector,omitempty"`
 }
 
+// AdditionalFormats specifies any additional formats to write to the configmap
+type AdditionalFormats struct {
+	Jks *KeySelector `json:"jks,omitempty"`
```

Comment:
nitpick: As mentioned in our call, I'd probably choose to have the Golang field be `JKS` and to keep the JSON key as `jks`.

Reply:
I think I've renamed this everywhere appropriate